matchms/Pipeline.py

```diff
@@ -378,28 +376,16 @@ def check_files_exist(filenames):
 
     # Getter & Setters
     @property
-    def logging_file(self):
-        return self.workflow["logging"].get("logging_file")
-
-    @logging_file.setter
-    def logging_file(self, file):
-        self.workflow["logging"]["logging_file"] = file
```

Comment:
@florian-huber My motivation was not so much the number of lines of code.

The setter/getter for logging was removed since it is not an attribute of workflow anymore, instead, it is now an attribute of Pipeline, so the setter and getter were now redundant. 

In addition, the setters were removed for all workflow settings, since I did not want it to be possible to change a workflow after a pipeline object is created. Since the SpectrumProcessor objects are initialized in __init__ of workflow, the problem with later changing anything in workflow is that the SpectrumProcessor objects (self.processing_queries) will not be updated. Creating unexpected behavior when running pipeline. 

A solution could be still creating setters, that after setting call _initialize_spectrum_processor_queries. That would look like:
```python
@query_filters.setter
def query_filters(self, query_filters):
    self.workflow["query_filters"] = query_filters
    self._initialize_spectrum_processor_queries
```

Reply:
Will be implemented in #483